src/Filters/FiltersPartial.php

```diff
@@ -4,10 +4,14 @@
 
 use Illuminate\Database\Eloquent\Builder;
 
-class FiltersPartial implements Filter
+class FiltersPartial extends FiltersExact implements Filter
```

Comment:
Making `FiltersPartial` extend `FiltersExact` feels a bit dirty as not every partial filter is an exact filter...
I'll probably refactor this to make `FiltersPartials` and `FiltersExact` both extend a base `Filter` class or use a `FiltersRelatedModels` trait.